android/build.py

```diff
@@ -60,8 +61,11 @@
     '-Dandroid_ndk=' + ndk_path,
     '-Dandroid_abi=' + android_abi,
     '-Dandroid_strip=' + toolchain.strip,
+    '-Dopenssl:asm=disabled'
```

Comment:
This argument is usually needed on m1 based macs since they don't have `asm`.
I wasn't sure if there was a good way to disable building openssl at all with the meson wrap system so we could just drop this.